### ISSUE TITLE
Hyper combined with Icon Picker creates constant "Showing your unsaved changes" message

### ISSUE THREAD

**Issue by JeroenJRP** [verbb/hyper#216]
### Describe the bug

While #189 seems to fix the issue with "Showing your unsaved changes" when linking entries, the problem persists when using a [verbb/icon-picker](https://github.com/verbb/icon-picker) field within Hyper, within a Matrix field.

But only under certain conditions. When the `Link` field is empty, or contains certain values like a `#`, or when the `Icon Picker` field is empty, the problem does not present itself. But when an icon is selected and a full URL or partial URL like `/` are used in the `Link` field (or an Element is selected) then the problem does present itself.
There also need to be 2 Hyper fields with a Icon Picker field present for the problem to occur.

The problem does not occur when using the Icon Picker field  by itself or directly in a Matrix field, which makes me think this is a problem with Hyper and not Icon Picker, but I might be wrong about that.

### Steps to reproduce

1. Create a Matrix field with a Hyper field containing a Icon Picker field.
2. Add the field to a Section's Entry Type
3. Create an Entry using this Entry type, add 2 blocks to the Matrix field, setting Hyper's Link field to an URL i.e. `https://verbb.io/` or `/`, and select an icon in the Icon Picker field.
4. Save the Entry

In my test the Icon Picker plugin (v3.0.2) is set up with a `SVG Sprite` icon set

### Craft CMS version

5.5.3

### Plugin version

2.2.1

### Multi-site?

No

### Additional context

https://github.com/user-attachments/assets/f1bcd40d-c65d-4047-a231-2f1526c6c6f1

**Comment by engram-design** [comment]
I actually think this might be entirely unrelated to Icon Picker, and stems from the fact that the slashes in a URL are being escaped incorrectly.

For example:
```
// saved to the database
http:\/\/hyper-craft5.test\/admin

// generated in the fields front-end
http://hyper-craft5.test/admin
```

Which reflects what you're seeing. Should be fixed for the next release. To get this early, run `composer require verbb/hyper:"dev-craft-5 as 2.2.1"`.

**Comment by JeroenJRP** [comment]
Amazing, that seems to do it! Thanks!

**Comment by engram-design** [comment]
Fixed in [2.2.2](https://github.com/verbb/hyper/releases/tag/2.2.2)
